src/widgets/_views.scss

```diff
@@ -18,6 +18,12 @@ scrolledwindow {
     background-color: bg-color(1);
     color: $fg-color;
 
+    &:selected {
+        &:focus, & {
```

Comment:
This selector doesn't really make sense to me. Can you explain what you're trying to do?

Reply:
If X is X:selected:focus or X:focus, or :selected X.

Reply:
I'll rewrite this to be more concise